Histogram.php

```diff
@@ -4,13 +4,19 @@
 
 class Histogram extends Metric
 {
-    function observe($value)
+    function observe($value, $buckets = [])
     {
+        $attrs = [
+            'value' => $value,
+            'method' => 'observe',
+        ];
+
+        if (!empty($buckets)) {
```

Comment:
you can always set the buckets. If they are empty `[]` the default will be applied on the server (pushprom) side. Less code, more happy.